students/14th/team_6/hyun/project_westagram/user/views.py

```diff
@@ -0,0 +1,76 @@
+from django.views import View
+from .models import User
+from django.http import JsonResponse
+import json
+
+class SignUpView(View):
+    def post(self,request):
+        signup_data = json.loads(request.body)
+        try:
+
+            if User.objects.all().filter(
+                user_name = signup_data["name"]).exists():
+                return JsonResponse(
+                { 'message: ' : 'ID already exists'}, status = 400)
+            elif User.objects.all().filter(
+                user_email = signup_data["email"]).exists():
+                return JsonResponse(
+                { 'message: ' : 'Email already exists'}, status = 400)
+            elif User.objects.all().filter(user_phone_number =
+                    signup_data["phone_number"]).exists():
```

Comment:
```suggestion
            elif User.objects.all().filter(user_phone_number = signup_data["phone_number"]).exists():
```
`=`으로 줄을 구분하면 오히려 가독성이 떨어질 수 있기 때문에 한 줄로 합치시는 것을 추천드립니다